File: app/src/main/java/com/irishinterest/irishinterest/model/user/ContactUsMessageEncrypted.java
package com.irishinterest.irishinterest.model.user;


import com.fasterxml.jackson.annotation.JsonAnyGetter;
import com.fasterxml.jackson.annotation.JsonAnySetter;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

import java.util.HashMap;
import java.util.Map;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder({
        "contactName",
        "contactEmail",
        "contactMessage"
})
public class ContactUsMessageEncrypted {

    @JsonProperty("contactName")
    private String contactName;
    @JsonProperty("contactEmail")
    private String contactEmail;
    @JsonProperty("contactMessage")
    private String contactMessage;
    @JsonIgnore
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    @JsonProperty("contactName")
    public String getContactName() {
        return contactName;
    }

    @JsonProperty("contactName")
    public void setContactName(String contactName) {
        this.contactName = contactName;
    }

    @JsonProperty("contactEmail")
    public String getContactEmail() {
        return contactEmail;
    }

    @JsonProperty("contactEmail")
    public void setContactEmail(String contactEmail) {
        this.contactEmail = contactEmail;
    }

    @JsonProperty("contactMessage")
    public String getContactMessage() {
        return contactMessage;
    }

    @JsonProperty("contactMessage")
    public void setContactMessage(String contactMessage) {
        this.contactMessage = contactMessage;
    }

    @JsonAnyGetter
    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    @JsonAnySetter
    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: app/src/main/java/com/irishinterest/irishinterest/model/user/ContactUsMessageEncrypted.kt
package com.irishinterest.irishinterest.model.user

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonPropertyOrder
import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonIgnore
import com.fasterxml.jackson.annotation.JsonAnyGetter
import com.fasterxml.jackson.annotation.JsonAnySetter
import java.util.HashMap

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder("contactName", "contactEmail", "contactMessage")
class ContactUsMessageEncrypted {
    @get:JsonProperty("contactName")
    @set:JsonProperty("contactName")
    @JsonProperty("contactName")
    var contactName: String? = null

    @get:JsonProperty("contactEmail")
    @set:JsonProperty("contactEmail")
    @JsonProperty("contactEmail")
    var contactEmail: String? = null

    @get:JsonProperty("contactMessage")
    @set:JsonProperty("contactMessage")
    @JsonProperty("contactMessage")
    var contactMessage: String? = null

    @JsonIgnore
    private val additionalProperties: MutableMap<String, Any> = HashMap()
    @JsonAnyGetter
    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    @JsonAnySetter
    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}